modules/custom/inope/inope.install

```diff
@@ -0,0 +1,32 @@
+<?php
+/**
+ * @file
+ * Install file for iNope.
+ */
+
+/**
+ * Enables the iNope Basic Configuration and Features.
+ */
+function inope_install() {
+  module_enable(array(
```

Comment:
I need to check really quick - but I think you should be able to utilize the master module for these enables via aquifer. If so I'll report right back about that.